### PR TITLE
NR-80059-MERN-stack-integration

### PR DESCRIPTION
**Pull request by pkosanam**
JIRA Tickets
1. MERN stack: https://issues.newrelic.com/browse/NR-80059

Description:
This code change includes MERN quickstart pages

Preview:
screencapture-newrelic-instant-observability-preview-2023-02-13-21_37_21

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by brammerl** [comment]
Hello, We are no longer accepting install plans as a part of the quickstart schema. We are instead moving to use data sources. You can either refer to an [already existing data source](https://github.com/newrelic/newrelic-quickstarts/tree/main/data-sources) in this repository or[ make your own](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1) community data source. 

If this applies to your PR, please make the proper update to your quickstart's config to reference only data sources.

**Comment by pkosanam** [comment]
> Hello, We are no longer accepting install plans as a part of the quickstart schema. We are instead moving to use data sources. You can either refer to an [already existing data source](https://github.com/newrelic/newrelic-quickstarts/tree/main/data-sources) in this repository or[ make your own](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1) community data source.
> 
> If this applies to your PR, please make the proper update to your quickstart's config to reference only data sources.

Thank you @brammerl. I have updated this PR by adding datasources files. Could you please review it and guide me if any changes are required.
Thank you 😊

**Comment by seemantk** [comment]
Keeping this open as the docs for this are almost ready

File: alert-policies/mexn/FirstContentfulPaint.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Name of the alert
+name: First Contentful Paint
+
+# Description and details
+description: |+
+  This alert is triggered when the first contentful paint is taking more than 3 seconds for 5 minutes.
```
Comment (seemantk) [inline alert-policies/mexn/FirstContentfulPaint.yml:6]: is taking -> takes

File: alert-policies/mexn/MongoDBConnectionsCount.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: MongoDB Collection Count
```
Comment (seemantk) [inline alert-policies/mexn/MongoDBConnectionsCount.yml:2]: Is this supposed to be MongoDB Connection Count?

File: alert-policies/mexn/MongoDBConnectionsCount.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+# Description and details
+description: |+
+  This alert is triggered when the created MongoDB connections are higher than 2000 for 5 minutes.
```
Comment (seemantk) [inline alert-policies/mexn/MongoDBConnectionsCount.yml:6]: Please consider rephrasing to:
This alert is triggered when the there are more than 2000 MongoDB connections created within a 5 minute period.

File: alert-policies/mexn/TransactionErrors.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: Transaction Errors
+
+description: |+
+  This alert is triggered when more than 10 transactions fail for 5 minutes.
```
Comment (seemantk) [inline alert-policies/mexn/TransactionErrors.yml:4]: more than 10 -> 10 or more

File: dashboards/mexn/dashboard.json [hunk 5]
```diff
@@ -0,0 +1,971 @@
...
+                "query": "SELECT percentile(duration, 50) FROM BrowserInteraction WHERE (`mobileOptimized` IS NULL) AND (category IN ('Initial page load', 'Route change')) LIMIT 1000 FACET CASES (WHERE category LIKE '%Initial page load%' AS 'Initial Page Load (Window load + AJAX)', WHERE category LIKE '%Route change%' AS 'Route Change') TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Ajax Request",
```
Comment (seemantk) [inline dashboards/mexn/dashboard.json:99]: Request -> Requests

**Review by seemantk — CHANGES_REQUESTED** [review]
Nice work here @pkosanam .  Added some language adjustments to increase clarity

**Comment by pkosanam** [comment]
Thank you for your review comments @seemantk
I have updated the code as per your review comments.
Thank you 😃

**Comment by brammerl** [comment]
Blocks #1598 as it contains a dashboard that #1598 references

File: data-sources/mern/config.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+id: mern-stack
+displayName: MERN Stack Integration
+description: |
+  ## Comprehensive MERN stack monitoring system:
```
Comment (brammerl) [inline data-sources/mern/config.yml:4]: Data source descriptions don't support markdown and are expected to be a short description. Please come up with a short/concise description for this data source

File: quickstarts/mern/config.yml [hunk 7]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (brammerl) [inline quickstarts/mern/config.yml:1]: Let's remove this space

File: quickstarts/mern/config.yml [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+# Name of the quickstart
+slug: mern-stack
+
+# Title of the quickstart
+title: MERN stack
```
Comment (brammerl) [inline quickstarts/mern/config.yml:6]:
```suggestion
title: MERN Stack
```

File: quickstarts/mern/config.yml [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+authors:
+  - New Relic
+  - Phaneendra Kosanam
+
+installPlans:
+  - mongodb-integration
+  - node-agent
+  - browser-docs
+
+keywords:
```
Comment (brammerl) [inline quickstarts/mern/config.yml:44]: If you want your quickstart to appear under a certain category in add data, please add specific keywords that will map to a specific category. The terms that appear match to the displayName.
[Query](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

**Review by seemantk — APPROVED** [review]

**Pull request merged**